packages/labs/observers/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+export * from './mutation_controller.js';
```

Comment:
do we want an index file for this package? The implementations are big enough, and they're not really inter-dependent so that they'd be used together, so we might want to require separate imports.

Reply:
I guess for convenience. Is there any reason not to? Just because people will mistakenly import too much?

Reply:
Yeah. I generally try avoid these import bundling modules because they usually just make more code load than necessary. If the individual modules are very likely to be used together it's more ok.

Reply:
And the package.json just omits module, main, and typings?

Reply:
Done.